lib/CXGN/Trial/ParseUpload/Plugin/TrialExcelFormat.pm

```diff
@@ -368,16 +371,6 @@ sub _validate_with_plugin {
         push @error_messages, "Cell A".$seen_plot_names{$r->uniquename}.": plot name already exists: ".$r->uniquename;
     }
 
-    # check for multiple plots at the same position
-    foreach my $key (keys %seen_plot_keys) {
-        my $plots = $seen_plot_keys{$key};
-        my $count = scalar(@{$plots});
-        if ( $count > 1 ) {
-            my @pos = split('-', $key);
-            push @error_messages, "More than 1 plot is assigned to the position row=" . $pos[0] . " col=" . $pos[1] . " plots=" . join(',', @$plots);
```

Comment:
Should this be removed? Just checking

Reply:
That one is awkward - it is not connected with any .xlsx functionality. Looks more like some problem with rebasing / branches. I will check why it is even there.